operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-secrets-config.yaml

```
@@ -67,14 +68,18 @@ spec:
               if [[ -n $SIG_KEY_DATA ]]; then
                 echo "Signing secret exists."
               else
+                # Delete secret/signing-secrets if already exists since by default cosign creates immutable secrets
+                kubectl delete secrets -n openshift-pipelines signing-secrets
```

```suggestion
                kubectl delete secrets -n openshift-pipelines signing-secrets --ignore-not-found=true
```

just in case the secret does not exist at all

---

done, thanks.

---

done, thanks.